units/proc-sys-fs-binfmt_misc.mount

```diff
@@ -7,7 +7,7 @@
 
 [Unit]
 Description=Arbitrary Executable File Formats File System
-Documentation=https://www.kernel.org/doc/Documentation/binfmt_misc.txt
+Documentation=https://www.kernel.org/doc/Documentation/admin-guide/binfmt-misc.rst
```

Comment:
uh, an 'rst' file? isn't there maybe a properly readable version of this somewhere?